announcementsAPI.json

```diff
@@ -288,6 +288,58 @@
         }
       }
     },
+    "/announcements/like": {
+      "put": {
+        "tags": [
+          "AnnouncementsEndpoints"
+        ],
+        "summary": "Add announcement to liked announcements for adopter provided in token. Requires adopter role.",
```

Comment:
Looking at the JSON structure, this endpoint is responsible for both adding to and removing from liked announcements. If so, then the description, in my opinion, is quite misleading.

Reply:
Yes, you're right. I will make change in the description